src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/types.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+export interface DataNormalizer {
+    normalize(data: SelectionData): SelectionData,
+}
+
+export type SelectionData = {
+    left: number,
+    top: number,
+    width: number,
+    height: number,
+};
+
+export type RectangleChange = {
+    topChange: number,
+    leftChange: number,
+    widthChange: number,
+    heightChange: number,
```

Comment:
This is added because otherwise it could be confused with the SelectionData type.